pom.xml

```
@@ -64,7 +67,7 @@ under the License.
 		<mockito.version>3.12.4</mockito.version>
```

```suggestion
		<mockito.version>5.14.2</mockito.version>
```
as for flink 2.0, in case it is not that hard